recipes/giac/patches/nofltk-check.patch

```diff
@@ -1,285 +0,0 @@
-+N:=display(normal(isobarycenter(T2)),536870968) ;
-+O:=display(normal(isobarycenter(T3)),536870968) ;
- MN:=color(segment(M,N),red) ;
- NO:=color(segment(N,O),red) ;
- OM:=color(segment(O,M),red) ;
-@@ -70,18 +70,18 @@
- nodisp(print("Feuerbach theorem")) ;
- assume(xa=(1/2)) ;
- assume(ya=1) ;
--A:=point(xa,ya) ;
--B:=point(-1,0) ;
--AB:=segment(A,B) ;
--C:=point(1,0) ;
--AC:=segment(A,C) ;
--CB:=segment(C,B) ;
--E:=midpoint(AB) ;
--F:=midpoint(AC) ;
--G:=midpoint(CB) ;
--c:=circumcircle(E,F,G) ;
--H:=altitude(A,B,C) ;
--I:=(inter(H,CB))[0] ;
-+A:=point(xa,ya,display=536870968) ;
-+B:=point(-1,0,display=536870968) ;
-+AB:=segment(A,B,color=56) ;
-+C:=point(1,0,display=536870968) ;
-+AC:=segment(A,C,color=56) ;
-+CB:=segment(C,B,color=56) ;
-+E:=color(midpoint(AB),56) ;
-+F:=color(midpoint(AC),56) ;
-+G:=color(midpoint(CB),56) ;
-+c:=circumcircle(E,F,G,color=56) ;
-+H:=altitude(A,B,C,color=56) ;
-+I:=(inter(H,CB,color=56))[0] ;
- nodisp(print("teste que le pied de la altitude est sur c de 2 facons, doit renvoyer 1 puis 0")) ;
- is_element(I,c) ;
- nodisp(O:=center(c)) ;
-@@ -90,7 +90,7 @@
- normal(l2-rc2) ;
- nodisp(print("circle incircle de A,B,C, center")) ;
- nodisp(d:=incircle(A,B,C)) ;
--P:=center(d) ;
-+P:=color(center(d),56) ;
- OP:=distance2(O,P) ;
- rd:=radius(d) ;
- rd2:=normal(rd^2) ;
-@@ -104,17 +104,17 @@
- yb:=0 ;
- assume(xc=-1) ;
- assume(yc=1) ;
--A:=point(xa,ya) ;
--B:=point(xb,yb) ;
--C:=point(xc,yc) ;
--triangle(A,B,C) ;
--G:=circumcircle(A,B,C) ;
-+A:=point(xa,ya,display=536870968) ;
-+B:=point(xb,yb,display=536870968) ;
-+C:=point(xc,yc,display=536870968) ;
-+triangle(A,B,C,color=56) ;
-+G:=circumcircle(A,B,C,color=56) ;
- assume(x=1) ;
- assume(y=1) ;
--M:=point(x,y) ;
--P:=projection(line(B,C),M) ;
--Q:=projection(line(C,A),M) ;
--R:=projection(line(A,B),M) ;
-+M:=point(x,y,display=536870968) ;
-+P:=projection(line(B,C),M,color=56) ;
-+Q:=projection(line(C,A),M,color=56) ;
-+R:=projection(line(A,B),M,color=56) ;
- D1:=factor(numer(im((Q-P)*conj(R-P)))) ;
- D12:=normal(-D1/yc^2) ;
- D2:=equation(G) ;
-@@ -124,8 +124,8 @@
- nodisp(print("Morley theorem")) ;
- assume(a1=0.3) ;
- assume(a2=0.4) ;
--A:=point(0) ;
--B:=point(1) ;
-+A:=point(0,display=536870968) ;
-+B:=point(1,display=536870968) ;
- nodisp(C:=point(texpand((tan(a2*3))/(tan(a1*3)+tan(a2*3))*(1+(i)*tan(a1*3))))) ;
- nodisp(P:=normal(point(texpand((tan(a2))/(tan(a1)+tan(a2))*(1+(i)*tan(a1)))))) ;
- nodisp(R:=normal(single_inter(line(0,1+(i)*texpand(tan(2*a1))),line(C,C+1+(i)*texpand(tan(pi/3+2*a1-a2)))))) ;
-diff -ruN check/TP00-sol.cas check.new/TP00-sol.cas
---- a/check/TP00-sol.cas	2014-07-02 15:06:55.000000000 +0200
-+++ b/check/TP00-sol.cas	2015-03-02 15:43:16.000000000 +0100
-@@ -67,7 +67,7 @@
- a:=exp(2*I*Pi/n);
- S:=seq(point(a^i,display=point_width_3+blue),i=1..n):
- /* Un cercle  passe par 3 sommets ssi  il coincide avec le cercle �unite. Donc si l'angle au centre n'est pas multiple de (2IPi/n) c'est bon. */
```

Comment:
`£` is here. I'll try manually editing this patch to avoid this line and the 2 others.